commands/announce_events_happening_today.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Every morning, announce upcoming events for the day, and
+ * events that are due tomorrow.
+ *
+ * @author Yaw Boakye <wheresyaw@gmail.com>
+ */
+'use strict';
+
+const config = require('../config');
+const getEventsHappeningToday = require('../helpers').getEventsHappeningToday;
```

Comment:
Won't destructuring work for these? Just curious.

`const {getEventsHappeningToday} = require('../helpers');`